components/meter/meter-circle.js

```diff
@@ -0,0 +1,61 @@
+import '../colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { bodyStandardStyles } from '../typography/styles.js';
+
+class MeterCircle extends LitElement {
+	static get properties() {
+		return {
+			value: { type: Number },
+			max: { type: Number }
+		};
+	}
+
+	static get styles() {
+		return [ bodyStandardStyles, css`
+		:host {
+			display: inline-block;
+			width: 2.4rem;
+		}
+		.d2l-meter-radial-full-bar,
+		.d2l-meter-radial-progress-bar {
+			fill: none;
+			stroke-linecap: round;
+			stroke-width: 6;
+		}
+		.d2l-meter-radial-full-bar {
+			stroke: var(--d2l-color-gypsum);
+		}
+		.d2l-meter-radial-progress-bar {
+			stroke: var(--d2l-color-celestine);
+		}
+		.d2l-meter-radial-text {
+			fill: var(--d2l-color-ferrite);
+			font-size: 0.6rem;
+			line-height: 0;
+		}
+	` ];
+	}
+
+	render() {
+		const lengthOfLine = 21 * 3.14 * 2; // approximation perimeter of circle
```

Comment:
If you want something more precise: [Math.PI](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/PI)

Reply:
I don't think we need the precision but for readability this is probably way better.